easybuild/easyconfigs/c/cURL/cURL-7.46.0-iomkl-2016.07.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.46.0'
+
+homepage = 'http://curl.haxx.se'
+description = """libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS,
+ POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP. libcurl supports
+ SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form based upload,
+ proxies, cookies, user+password authentication (Basic, Digest, NTLM, Negotiate,
+ Kerberos), file transfer resume, http proxy tunneling and more."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.07'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://curl.haxx.se/download/']
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# dependencies = [('OpenSSL', '1.0.1p')]
+# configopts = "--with-ssl=$EBROOTOPENSSL"
+
+modextravars = {'CURL_INCLUDES': '%(installdir)s/include'}
+
+sanity_check_paths = {
+    'files': ['bin/curl', 'lib/libcurl.a', 'lib/libcurl.so'],
```

Comment:
@edmondac please replace `'lib/libcurl.so'` with `'lib/libcurl.%s' % SHLIB_EXT` (in all cURL easyconfigs in this PR)